apps/meteor/client/views/account/preferences/PreferencesUserPresenceSection.tsx

```diff
@@ -27,14 +30,19 @@ const PreferencesUserPresenceSection = () => {
 					</FieldRow>
 				</Field>
 				<Field>
-					<FieldLabel htmlFor={idleTimeLimit}>{t('Idle_Time_Limit')}</FieldLabel>
+					<FieldLabel htmlFor={idleTimeLimitH}>{t('Idle_Time_Limit')}</FieldLabel>
```

Comment:
don't you have access to a OnChange callback or smth?
so you can update idleTimeLimit without needing to have 3 separate settings